cmd/integration/integration.go

```diff
@@ -915,7 +915,7 @@ func main() {
 
 	go func() {
 		defer util.FlushLogs()
-		time.Sleep(3 * time.Minute)
+		time.Sleep(5 * time.Minute)
```

Comment:
picked up by an earlier rebase, accidentally included as part of this PR; will remove